docs/02-app/01-building-your-application/03-rendering/01-partial-prerendering.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Partial Prerendering (Experimental)
+description: Learn how to combine the benefits of static and dynamic rendering with Partial Prerendering.
+---
+
+> Partial Prerendering is an **experimental** feature and subject to change.
+
+Partial Prerendering (PPR) is a rendering strategy that combines the benefits of static and dynamic rendering on the same route.
+
+Next.js currently defaults to static rendering unless you use [dynamic functions](/docs/app/building-your-application/rendering/server-components#dynamic-functions). These APIs opt a whole route into dynamic rendering.
```

Comment:
I got tripped up here. I was expecting to read about PPR and then got confused that this was about the non-PPR case. It's possible that, without prior knowledge, I wouldn't have realized what you meant.

Variations that may have helped me:
- Switching the order of this and the next paragraph?
- Leading this paragraph with "Without PPR, Next.js currently..."

Reply:
Thank you for the feedback, will rework it a bit 👍🏼